package.json

```diff
@@ -34,20 +34,21 @@
   ],
   "devDependencies": {
     "chai": "^3.5.0",
-    "eslint": "^4.3.0",
+    "eslint": "^4.19.1",
     "eslint-config-screwdriver": "^3.0.0",
     "jenkins-mocha": "^4.0.0",
     "mockery": "^2.0.0",
     "sinon": "^2.3.4"
   },
   "dependencies": {
-    "circuit-fuses": "^3.0.0",
+    "circuit-fuses": "^3.3.0",
     "cron-parser": "^2.5.0",
     "hoek": "^5.0.3",
-    "ioredis": "^3.1.2",
-    "node-resque": "^4.0.7",
-    "screwdriver-executor-base": "^6.1.0",
+    "ioredis": "^3.2.2",
+    "node-resque": "^4.0.9",
```

Comment:
do you need to pull in node-resque `^5.3.0` like in https://github.com/screwdriver-cd/queue-worker/pull/60?

Reply:
I'm thinking of pulling this one in BEFORE the queue-worker.
Don't think it will affect anything if passing in extra params to worker.